Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -161,27 +161,27 @@
  ################
  # BATH'S BATHS #
  ################
- building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+ building_type_btwk_aquae_sulis_baths_01:0 "Baths of Aquae Sulis"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans occupied and developed a British shrine into a resort. The shrine (dedicated to the god #italic Sulis#!) had been built around a series of natural springs, later influencing the area's Latin name - #italic Aquae Sulis#!. These springs would later provide the foundations for the magnificent bathing complex Aquae Sulis is famed for."
 
- building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
- building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
- building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
+ building_btwk_aquae_sulis_baths_01:0 "Abandoned Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nUnfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. Many locals now instead opt for bathing in the natural springs themselves." 
+ building_btwk_aquae_sulis_baths_02:0 "Restored Baths"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nLeft to ruin since the Empire's departure, the bathing complex has since been completely renovated. Men of all manners of life now gather in the communal hub, like citizens of the Rome of old, with both locals and travellers from further afield imparting their custom to the province - a great boon to its economy to be sure."
```

Comment:
the bathing complex has since been completely renovated -> the baths have now been completely renovated and restored to their former glory.
Men of all manners of life now gather in the communal hub, like citizens of the Rome of old, with both locals and travellers from further afield imparting their custom to the province - a great boon to its economy to be sure -> The baths now serve as a cultural centre and communal hub for the region where men from all manners of life can gather, like the Roman citizens of old. (Not really sure about the last part you had but you can keep it if you want)